src/core/views/components/iframe.js

```diff
@@ -39,6 +39,9 @@ class Iframe extends Component {
         style={{ border: 0, ...style }}
         title='typeform-embed'
         width='100%'
+        mozallowfullscreen='mozallowfullscreen'
+        webkitallowfullscreen='webkitallowfullscreen'
+        allowfullscreen='allowfullscreen'
```

Comment:
[Those attributes are deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-allowfullscreen). You can add permission to [embed-permissions.js](https://github.com/Typeform/embed/blob/master/src/core/utils/embed-permissions.js#L1).